src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultRazorDynamicFileInfoProvider.cs

```diff
@@ -329,10 +329,7 @@ private void ProjectManager_Changed(object? sender, ProjectChangeEventArgs args)
 
     private ProjectId? TryFindProjectIdForProjectKey(ProjectKey key)
     {
-        if (_projectSnapshotManagerAccessor.Instance.Workspace is not { } workspace)
```

Comment:
💭 Interestingly, it was never possible for `Workspace` to be null. So, this check and others weren't necessary.